### PR TITLE
JS version of this action

### PR DESCRIPTION
**Pull request by marcocarvalho**
**Description**: Move to a JS version to avoid docker build steps on the action. It will run faster and open the possibility to expand this action to be completely automatic, without the need for `reruns`.

**JIRA**: NO_JIRA

**How to Verify**: Test it on a project

### CODE REVIEW

**Comment by marcocarvalho** [comment]
Testing in https://github.com/CareMessagePlatform/monitoring-app/pull/495

image

**Comment by marcocarvalho** [comment]
After merged to staging: 

image

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+version: 2
```
Comment (marcocarvalho) [inline .circleci/config.yml:1]: I setup the circleci build to run the tests, but it's missing the deployment key to be able to checkout the project, can you help me with this @raivil or only Daniel can do it?

File: action.yml [hunk 2]
```diff
@@ -13,16 +13,10 @@ inputs:
...
+    required: true
+    default: ${{ github.token }}
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
-  args:
-    - ${{ inputs.destination_branch }}
-    - ${{ inputs.source_branch }}
+  using: 'node16'
+  main: 'dist/index.js'
```
Comment (marcocarvalho) [inline action.yml:22]: Now it takes 1 to 2 seconds to run this action :)

File: .circleci/config.yml (hunk 1 above)
Comment (raivil) [reply .circleci/config.yml:1]: @marcocarvalho I started "following" the repository on CircleCI and it looks like it added a deploy key automatically on GitHub.
```
Added on Jan 18, 2022 by CircleCI with authorization from @raivil
Never used — Read-only
```

Re-running the project works. ✨

**Review by raivil — DISMISSED** [review]

**Pull request merged**